erpnext/stock/doctype/stock_entry/stock_entry.js

```diff
@@ -618,6 +618,12 @@ erpnext.stock.StockEntry = erpnext.stock.StockController.extend({
 			};
 		});
 
+		if(this.frm.fields_dict["items"].grid.get_field('package_tag')) {
```

Comment:
You don't need the `if` check here, we know there's both the `items` and `package_tag` fields